lib/indexer/compare_enumerator.rb

```diff
@@ -74,7 +74,7 @@ def client
     end
 
     def search_config
-      @search_config ||= SearchConfig.new
+      @search_config ||= Rummager.search_config
```

Comment:
Would this need to revert considering how we got stuff working for the `govuk_index` PR?